### PR TITLE
Update the bootstrap variables and imports, because bootstrap version…

### PR DESCRIPTION
**Pull request by Katrienvh**
…s is updated

## Type
- Enhancement
- Feature

## Pull request description
There were al lot of new variables that bootstrap was using, but those where not in our variables file

### CODE REVIEW

File: src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss [hunk 1]
```diff
@@ -26,17 +22,21 @@ $gray-900: #212529;
...
-  "200": $gray-200,
-  "300": $gray-300,
-  "400": $gray-400,
-  "500": $gray-500,
-  "600": $gray-600,
-  "700": $gray-700,
-  "800": $gray-800,
-  "900": $gray-900
-), $grays);
+// stylelint-disable-next-line scss/dollar-variable-default
```
Comment (sarahmey) [inline src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss]: what is this doing? (in multiple places)

File: src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss [hunk 2]
```diff
@@ -76,40 +80,46 @@ $light:         $gray-100;
...
+    ),
+    $theme-colors
+);
 
 // Set a specific jump point for requesting color jumps
 $theme-color-interval:      8%;
 
 // The yiq lightness value that determines when the lightness of color changes from "dark" to "light". Acceptable values are between 0 and 255.
-$yiq-contrasted-threshold: 150;
+$yiq-contrasted-threshold:  150;
```
Comment (sarahmey) [inline src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss]: too much space

File: src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss [hunk 3]
```diff
@@ -76,40 +80,46 @@ $light:         $gray-100;
...
 
 // Customize the light and dark text colors for use in our YIQ color contrast function.
-$yiq-text-dark: $gray-900;
-$yiq-text-light: $white;
+$yiq-text-dark:             $gray-900;
+$yiq-text-light:            $white;
+
```
Comment (sarahmey) [inline src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss]: enter too much

**Comment by carakas** [comment]
@Katrienvh can you take a look at the feedback

**Comment by Katrienvh** [comment]
Closing because outdated

**Pull request closed without merging**